ts/components/core/variables/IOStyles.ts

```diff
@@ -23,6 +23,11 @@ export const IOStyles = StyleSheet.create({
   rowSpaceBetween: {
     flexDirection: "row",
     justifyContent: "space-between"
+  },
+  //https://github.com/pagopa/io-app/pull/4387
+  topBorderForMessagesAndServices: {
```

Comment:
Instead of referring to a specific section in the name, can we reference the effect that we'd like to achieve? Like "innerComponentStyleForComponentsThatAreInsideATabNavigatorInOrderForTheTopBorderNotToBeVisible" but under 32 characters?